stdlib/src/hashlib.rs

```diff
@@ -229,90 +231,142 @@ mod hashlib {
         }
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_md5")]
     fn md5(args: HashArgs) -> PyHasher {
         PyHasher::new("md5", HashWrapper::new::<Md5>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha1")]
     fn sha1(args: HashArgs) -> PyHasher {
         PyHasher::new("sha1", HashWrapper::new::<Sha1>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha224")]
     fn sha224(args: HashArgs) -> PyHasher {
         PyHasher::new("sha224", HashWrapper::new::<Sha224>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha256")]
     fn sha256(args: HashArgs) -> PyHasher {
         PyHasher::new("sha256", HashWrapper::new::<Sha256>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha384")]
     fn sha384(args: HashArgs) -> PyHasher {
         PyHasher::new("sha384", HashWrapper::new::<Sha384>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha512")]
     fn sha512(args: HashArgs) -> PyHasher {
         PyHasher::new("sha512", HashWrapper::new::<Sha512>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha3_224")]
     fn sha3_224(args: HashArgs) -> PyHasher {
         PyHasher::new("sha3_224", HashWrapper::new::<Sha3_224>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha3_256")]
     fn sha3_256(args: HashArgs) -> PyHasher {
         PyHasher::new("sha3_256", HashWrapper::new::<Sha3_256>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha3_384")]
     fn sha3_384(args: HashArgs) -> PyHasher {
         PyHasher::new("sha3_384", HashWrapper::new::<Sha3_384>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_sha3_512")]
     fn sha3_512(args: HashArgs) -> PyHasher {
         PyHasher::new("sha3_512", HashWrapper::new::<Sha3_512>(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_shake_128")]
     fn shake_128(args: HashArgs) -> PyHasherXof {
         PyHasherXof::new("shake_128", HashXofWrapper::new_shake_128(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_shake_256")]
     fn shake_256(args: HashArgs) -> PyHasherXof {
         PyHasherXof::new("shake_256", HashXofWrapper::new_shake_256(args.string))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_blake2b")]
     fn blake2b(args: BlakeHashArgs) -> PyHasher {
         PyHasher::new("blake2b", HashWrapper::new::<Blake2b512>(args.data))
     }
 
-    #[pyfunction]
+    #[pyfunction(name = "openssl_blake2s")]
     fn blake2s(args: BlakeHashArgs) -> PyHasher {
         PyHasher::new("blake2s", HashWrapper::new::<Blake2s256>(args.data))
     }
 
-    trait ThreadSafeDynDigest: DynClone + DynDigest + Sync + Send {}
+    #[pyfunction]
+    fn compare_digest(
+        a: PyObjectRef,
+        b: PyObjectRef,
```

Comment:
```suggestion
        a: ArgStrOrBytesLike,
        b: ArgStrOrBytesLike,
```
This type will be helpful